helm/sealed-secrets/values.yaml

```diff
@@ -404,6 +404,22 @@ rbac:
   ## @param rbac.pspEnabled PodSecurityPolicy
   ##
   pspEnabled: false
+  ## "Proxier" RBAC Role configuration
+  ##
+  proxier:
+    ## @param create Specifies whether to create the "proxier" role, to allow external users to access the SealedSecret API
+    ##
+    create: true
+    ## @param bind Specifies whether to create a RoleBinding for the "proxier" role
+    ##
+    bind: true
```

Comment:
I'm not 100% sure about this. Are you thinking of some scenario where one would want to create the role but no rolebinding for it?

Reply:
say if it's used as a dependency chart, a situation could arise where it's easier to create the rolebinding in the parent chart, rather than passing values to the subchart. or maybe the subjects to bind aren't known in this context and some other helm release handles the actual binding. I don't directly need this use case, just seemed plausibly useful since delegating admin functions can get complex